docs/documentation/backwards-compatibility.md

```diff
@@ -0,0 +1,68 @@
+# Using backwards compatibility
+
+Version 7 of the Prototype Kit uses the new GOV.UK Design System. It is not compatible with prototypes built with older versions by default.
+
+Backwards compatibility lets you import a large old prototype into version 7, without having to rewrite it. You can update old pages one by one as you need to, and add new pages.
+
+You will end up with 2 'apps' in your prototype:
+
+**/app** using version 7
+
+**/app/v6** using version 6
+
+If any pages or routes exist in both apps, the one in version 7 will win.
+
+## Instructions
+
+1. Make a note of your Prototype Kit version in **VERSION.txt** in your prototype folder.
+
+1. Download the latest Prototype Kit and unzip it.
```

Comment:
Would it make sense for this step to come after "Copy everything from the new Prototype Kit folder into your prototype folder."? It would make it clearer that the steps before these are all for the user's old prototype.

Reply:
excellent idea